src/constants/translations/ua/constant.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
The name of the translation file should be changed. It is not a constants...

Reply:
Resolved